test/Test/Fission/Error.hs

```diff
@@ -0,0 +1,49 @@
+module Test.Fission.Error (tests) where
+
+import Test.Fission.Prelude as Mock
+import Fission.Error
+import Data.List as L
```

Comment:
A couple minor things:

Ideally use more than one character for legibility. `List.foo` is better than `L.foo` in almost all cases.

They're the same underneath, but for consistency, it probably best to keep to the RIO versions of everything

```haskell
import qualified RIO.List as List

List.length successEffectLog `shouldBe` 1
```

or in this case, the general [`length`](https://www.stackage.org/haddock/lts-16.2/base-4.13.0.0/Prelude.html#v:length) function from `base` (should be imported by default)

```haskell
length successEffectLog `shouldBe` 1
```